test/unit/core/dom/layout/test-layout.js

```diff
@@ -662,39 +662,36 @@ describes.realWin('ampshared.css', {amp: true}, function (env) {
     doc.body.appendChild(element);
   });
 
-  describe
-    .configure()
-    .enableIe()
-    .run('overflow', function () {
-      let overflow;
-
-      beforeEach(() => {
-        overflow = doc.createElement('div');
-        overflow.setAttribute('overflow', '');
-        overflow.style.height = '20px';
-        element.appendChild(overflow);
-      });
-
-      it('should not allow overflow element to distort a size-defined layout', () => {
-        element.setAttribute('layout', 'responsive');
-        element.setAttribute('width', 100);
-        element.setAttribute('height', 100);
-        expect(applyStaticLayout(element)).to.equal(Layout.RESPONSIVE);
-
-        expect(element.offsetWidth).to.equal(element.offsetHeight);
-        expect(overflow.offsetHeight).to.equal(20);
-        expect(win.getComputedStyle(overflow).position).to.equal('absolute');
-      });
-
-      it('should allow overflow element to distort container layout', () => {
-        element.setAttribute('layout', 'container');
-        overflow.text = 'test';
-        expect(applyStaticLayout(element)).to.equal(Layout.CONTAINER);
-
-        expect(element.offsetHeight).to.equal(overflow.offsetHeight);
-        expect(win.getComputedStyle(overflow).position).to.equal('relative');
-      });
+  describe.configure().run('overflow', function () {
```

Comment:
nit: `describe.configure().run(...)` can be simplified to just `describe(...)`, since there's no longer any custom configuration being applied.

This can be done for all three instances in this PR.